app/posts/govuk-services-frontend-stats.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Measuring how government services are using GOV.UK Frontend
+description: How I wrote a script to detect which versions of GOV.UK Frontend different government services are using
+date: 2023-12-20
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+image:
+  src: /assets/posts/govuk-services-frontend-stats/govuk-frontend-version-stats.png
+  alt: GOV.UK Frontend Version stats
+  opengraphImage: true  
```

Comment:
How about no illustration; not a requirement and not always needed/useful.

```suggestion
```

Reply:
Fair! I had half a mind to use them on the index pages in future - but can always come back and add them later if we decide to do that.